R/compute_metrics.R

```diff
@@ -0,0 +1,160 @@
+#' Calculate and format metrics from tuning functions
+#'
+#' This function computes metrics from tuning results. The arguments and
+#' output formats are closely related to those from [collect_metrics()], but
+#' this function additionally takes a `metrics` argument with a metric set
+#' for new metrics to compute. This allows for computing new performance
+#' metrics without requiring users to re-evaluate models against resamples.
+#'
+#' Note that the [control option][control_grid()] `save_pred = TRUE` must
+#' have been supplied when generating `x`.
```

Comment:
I think you should add a `@description` tag so it groups this `save_pred` sentence in the Description section rather than further down in Details, because it is somewhat important